llvm/lib/Target/RISCV/RISCVInstrInfoD.td

```diff
@@ -398,6 +398,7 @@ foreach Ext = DExts in {
   defm : PatFprFpr_m<fmaximumnum, FMAX_D, Ext>;
   defm : PatFprFpr_m<riscv_fmin, FMIN_D, Ext>;
   defm : PatFprFpr_m<riscv_fmax, FMAX_D, Ext>;
+  def : Pat<(f64 (fcanonicalize FPR64:$rs1)), (FMIN_D $rs1, $rs1)>;
```

Comment:
Does this mean this is still unsupported without an extension?

Reply:
It's the same extension that makes double precision FP a legal type.